[ERROR] 2016-11-22 22:58:28.692 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37305769
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:28.864 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37276647
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:28.949 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37279620
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:29.038 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37279654
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:29.135 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37279685
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:29.229 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37279702
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:29.321 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37279706
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:29.406 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37279711
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:29.494 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37287212
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:29.581 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37287510
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:29.669 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37287716
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:29.753 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37287812
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:29.836 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37288384
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:29.924 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37288782
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:30.011 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37289748
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:30.103 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37289778
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:30.189 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37289784
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:30.278 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37289840
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:30.366 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37289841
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:30.452 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37289868
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:30.541 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37289877
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:30.629 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37289881
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:30.710 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37289909
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:30.806 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37289919
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:30.895 [main] MysqlWriter - SQLException occurred while inserting comparable: primaryZPID = 37279691 and compZPID = 37289940
java.sql.SQLException: No value specified for parameter 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:957) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:896) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:885) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.checkAllParametersSet(PreparedStatement.java:2205) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2185) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.fillSendPacket(PreparedStatement.java:2115) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1936) ~[mysql-connector-java-5.1.38-bin.jar:5.1.38]
	at main.java.dbConnectors.MysqlWriter.insertObject(MysqlWriter.java:290) [classes/:?]
	at main.java.classes_for_db.ZillowComparable.writeToDB(ZillowComparable.java:233) [classes/:?]
	at main.java.execute.Executor.processComps(Executor.java:190) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:112) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 22:58:31.305 [main] GenericZillowAPICaller - IOException occured while making API call to: http://www.zillow.com/webservice/GetUpdatedPropertyDetails.htm?zws-id=X1-ZWz19thqfce13f_3s95g&zpid=37279691
org.apache.http.client.HttpResponseException: Internal Server Error
	at org.apache.http.impl.client.AbstractResponseHandler.handleResponse(AbstractResponseHandler.java:69) ~[httpclient-4.5.2.jar:4.5.2]
	at org.apache.http.client.fluent.Response.handleResponse(Response.java:90) ~[fluent-hc-4.5.2.jar:4.5.2]
	at org.apache.http.client.fluent.Response.returnContent(Response.java:97) ~[fluent-hc-4.5.2.jar:4.5.2]
	at main.java.api_calls.GenericZillowAPICaller.makeApiCall(GenericZillowAPICaller.java:38) [classes/:?]
	at main.java.api_calls.GenericZillowAPICaller.makeApiCall(GenericZillowAPICaller.java:49) [classes/:?]
	at main.java.execute.Executor.processPropertyDetails(Executor.java:149) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:113) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
[ERROR] 2016-11-22 23:01:19.739 [main] GenericZillowAPICaller - IOException occured while making API call to: http://www.zillow.com/webservice/GetUpdatedPropertyDetails.htm?zws-id=X1-ZWz19thqfce13f_3s95g&zpid=37279691
org.apache.http.client.HttpResponseException: Internal Server Error
	at org.apache.http.impl.client.AbstractResponseHandler.handleResponse(AbstractResponseHandler.java:69) ~[httpclient-4.5.2.jar:4.5.2]
	at org.apache.http.client.fluent.Response.handleResponse(Response.java:90) ~[fluent-hc-4.5.2.jar:4.5.2]
	at org.apache.http.client.fluent.Response.returnContent(Response.java:97) ~[fluent-hc-4.5.2.jar:4.5.2]
	at main.java.api_calls.GenericZillowAPICaller.makeApiCall(GenericZillowAPICaller.java:38) [classes/:?]
	at main.java.api_calls.GenericZillowAPICaller.makeApiCall(GenericZillowAPICaller.java:49) [classes/:?]
	at main.java.execute.Executor.processPropertyDetails(Executor.java:149) [classes/:?]
	at main.java.execute.Executor.getDataForAddress(Executor.java:113) [classes/:?]
	at main.java.execute.Executor.run(Executor.java:62) [classes/:?]
	at main.java.execute.Main.main(Main.java:55) [classes/:?]
